docs/cennznet/cennzx.md

```diff
@@ -19,103 +19,125 @@ The following sections contain the module details.
  
 # Storage
  
-### coreAssetId(): `AssetId`
+### coreAssetId(): `u32`
```

Comment:
Does this matter if `AssetId` has been switched back to the generic `u32`?

Reply:
https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/runtime/definitions.ts#L57
AssetId actually is u32.. This data comes from metadata